Dockerfile

```diff
@@ -15,6 +15,18 @@ RUN set -eux; \
     build-essential postgresql-client libmysqlclient-dev mysql-client python3.8 python3-pip python3.8-dev \
     uwsgi git libxml2-dev libxslt-dev zlib1g-dev
 
+# Install fuse and pachctl
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends curl fuse3\
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+RUN curl -o /tmp/pachctl.deb -L https://github.com/pachyderm/pachyderm/releases/download/v2.0.8/pachctl_2.0.8_amd64.deb\
```

Comment:
Make version configurable, eg expose it as `ENV PATCHCTL_VERSION=2.0.8`.

Delete deb package from tmp after install